### PR TITLE
Add "named parameters" example

### PR DESCRIPTION
**Pull request by MegaRedHand**

### CODE REVIEW

File: content/en/examples/named_parameters.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "named parameters"
+date: 2023-06-25T12:55:00-06:00
```
Comment (SantiagoPittella) [inline content/en/examples/named_parameters.md:3]: we are using `weight` now.

File: content/es/examples/named_parameters.md
Comment (SantiagoPittella) [inline content/es/examples/named_parameters.md:1]: The /es/** files MUST be in spanish.

**Review by SantiagoPittella — CHANGES_REQUESTED** [review]

File: content/en/examples/named_parameters.md (hunk 1 above)
Comment (MegaRedHand) [reply content/en/examples/named_parameters.md]: Fixed!

File: content/es/examples/named_parameters.md
Comment (MegaRedHand) [reply content/es/examples/named_parameters.md:1]: Translated!

**Review by Oppen — APPROVED** [review]

**Review by SantiagoPittella — APPROVED** [review]

**Pull request merged**